jekyll/_config.yml

```diff
@@ -98,8 +98,8 @@ collections:
     name: "2.0 Docs"
     site_section: true
     output: true
-    permalink: /2.0/:path/
```

Comment:
are we sure we don't need the slash in front?

Reply:
Everything seems to be working on my end

Reply:
I can add it back no worries and test